tools/integration_tests/run.py

```diff
@@ -50,16 +50,22 @@ def start_server():
     os.environ.setdefault("SYNC_MASTER_SECRET", "secret0")
     os.environ.setdefault("SYNC_CORS_MAX_AGE", "555")
     os.environ.setdefault("SYNC_CORS_ALLOWED_ORIGIN", "localhost")
-    os.environ['SYNC_TOKENSERVER__TEST_MODE_ENABLED'] = 'true'
+    mock_server_port = os.environ.get('MOCK_FXA_SERVER_PORT', 6000)
+    os.environ["SYNC_TOKENSERVER__FXA_BROWSERID_SERVER_URL"] = "http://localhost:%s/v2" % mock_server_port
+    os.environ["SYNC_TOKENSERVER__FXA_OAUTH_SERVER_URL"] = "http://localhost:%s" % mock_server_port
     the_server_subprocess = start_server()
     try:
         res = 0
         res |= run_live_functional_tests(TestStorage, sys.argv)
-        res |= run_local_tests()
+        os.environ.setdefault("TOKENSERVER_AUTH_METHOD", "oauth")
```

Comment:
We run all the local tests with both BrowserID and OAuth requests to be extra safe here